elide-core/src/main/java/com/yahoo/elide/parsers/state/RecordTerminalState.java

```diff
@@ -100,7 +100,7 @@ private JsonNode getResponseBody(PersistentResource rec, RequestScope requestSco
         JsonApiDocument jsonApiDocument = new JsonApiDocument();
 
         //TODO Make this a document processor
-        Data<Resource> data = rec == null ? null : new Data<>(rec.toResource());
+        Data<Resource> data = rec == null ? null : new Data<>(rec.toResourceWithSortingAndPagination());
```

Comment:
Do you think we need this?  I"m trying to figure out what this would mean for an individual entity.  Would this basically work for something like page 1, limit 0?